### PR TITLE
Base reply logic + prettyprint

### PR DESCRIPTION
**Pull request by SpencerMartel**
Bot can now reply in server, jsons can be read by humans!

### CODE REVIEW

File: steam.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
+from genericpath import exists
+from pickle import TRUE
```
Comment (Nwitz) [inline steam.py]: Uneeded imports, you're probably using the wrong true

File: steam.py [hunk 2]
```diff
@@ -106,7 +110,7 @@ def sync_wishlist_file():
     game_object_json = {'wishlist':wishlist_json}
 
     # Dump JSON object into String format and write string to file
-    game_list_string = json.dumps(game_object_json)
+    game_list_string = json.dumps(game_object_json, indent=4, sort_keys=TRUE)
```
Comment (Nwitz) [inline steam.py]:
```suggestion
    game_list_string = json.dumps(game_object_json, indent=4, sort_keys=True)

```

File: steam.py (hunk 1 above)
Comment (SpencerMartel) [reply steam.py]: no u

File: steam.py [hunk 3]
```diff
@@ -63,30 +66,31 @@ def add_game(app_tuple): # app_tuple = (appID, game name)
...
+    received_app_data = fetch_game_data(key)
+    if not 'price_overview' in received_app_data:
+        return (GameAddStatus.FREE_GAME, None)
     
-    app_data = fetch_game_data(key)
-    app_data['name'] = app_tuple[1]
-    print(json.dumps(app_data))
+    app_data = {'name' : app_tuple[1]}
+    app_data.update(received_app_data)
+    print(json.dumps(app_data, indent=4, sort_keys=TRUE))
```
Comment (Nwitz) [inline steam.py]:
```suggestion
    print(json.dumps(app_data, indent=4, sort_keys=True))

```

**Review by Nwitz — APPROVED** [review]
Nice

**Pull request merged**